relay/polkadot/src/lib.rs

```diff
@@ -367,6 +372,8 @@ impl pallet_mmr::Config for Runtime {
 	type WeightInfo = ();
 	type LeafData = pallet_beefy_mmr::Pallet<Runtime>;
 	type BlockHashProvider = pallet_mmr::DefaultBlockHashProvider<Runtime>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = ();
```

Comment:
Shouldn't this be better like this?
According to the stable2409 branch: https://github.com/paritytech/polkadot-sdk/blob/stable2409/polkadot/runtime/westend/src/lib.rs#L427

```suggestion
	type BenchmarkHelper = parachains_paras::benchmarking::mmr_setup::MmrSetup<Runtime>;
```